src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,11 +5,24 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String color;
+    private String model;
```

Comment:
Don't forget about access modifiers for your custom fields in `Bulldozer` class. It's a good practice to use private access modifiers and provide getters and setters for accessing these fields.